### ISSUE TITLE
AnalysisJob list view: Filtering

### ISSUE THREAD

**Issue by CloudNiner** [azavea/pfb-network-connectivity#139]
Filter list view table on:
- [ ] ~~django job id~~
- [ ] ~~batch job id~~
- [ ] ~~batch job status~~
- [ ] status
- [ ] neighborhood

**Comment by CloudNiner** [comment]
Depends on #94

**Comment by flibbertigibbet** [comment]
Currently has filters in place for neighborhood and status. Should this be changed to 'search by UUID or batch job UUID'? Would require implementing a text-field search that we currently do not have in the UI (just drop-down filtering).

**Comment by CloudNiner** [comment]
I just updated the original comment to remove the search on the django job and batch job uuids, since those are additional nice-to-haves that the client isn't specifically requesting. So if we have status and neighborhood filtering then this issue is done for now as well.

**Comment by flibbertigibbet** [comment]
Neighborhood filtering in place; status filtering will need to be updated since removed as filterable API field (do client-side?)

**Comment by CloudNiner** [comment]
While it might be pretty inefficient (since you'd have to load the entire queryset unless you did a custom sql query), it should be possible to do a custom method filter using django-filters. That's probably a better first shot than doing client side filtering, which could be awkward due to pagination as the job list gets large.

Example of the Django filters boilerplate here:
https://github.com/azavea/climate-change-api/blob/develop/django/climate_change_api/climate_data/filters.py#L23
